lib/active_admin/namespace_settings.rb

```diff
@@ -106,5 +106,8 @@ class NamespaceSettings < DynamicSettingsNode
 
     # Include association filters by default
     register :include_default_association_filters, true
+
+    # Switch between asset pipeline and webpacker assets
+    register :use_webpacker, false
```

Comment:
Can we instead toggle this on through an environment variable? I really appreciate the approach of enabling this in small steps, but I don't want to expose this kind of setting to end users yet since this is just a work in progress. Maybe set `ENABLE_ACTIVEADMIN_WEBPACKER_SUPPORT=true` in `config/boot.rb` of the generated application, and then check that when choosing to either use `_pack`, or `_link` helpers?

Reply:
Hum it does make sense to keep it private while working on it. But in this case, I would instead merge my work in a different order given #5865, and instead of merging in the order I developed it, I would merge it this way: 
- rework js files to remove the use of `ActiveAdmin` global constant that isn't necessary
- setup npm packaging and rollup (if rollup works for you of course, tell me if there another bundling option you'd prefer). JS assets would be compiled when pushing to the repo, and the generated bundle then used by sprockets
- release the npm package including assets ready to be imported by webpack
- add that `use_webpacker` switch when everything else is ready

Reply:
Sure, sorry I didn't yet provide feedback in the other PR. Your plan sounds good to me :+1:.

Reply:
starting with rollup in fact, as sprockets-es6 doesn't handle `imports`, see #5868